.github/workflows/release--build-hotfix.yml

```
@@ -21,3 +21,4 @@ jobs:
     with:
       base_ref: maint/v${{ github.event.inputs.releaseNumber }}
       tag_number: ${{ github.event.inputs.releaseVersion }}
+      push_tag_to_base_ref: true
```

```suggestion
      push_branch_to_base_ref: true
```

Ou alors "base_ref_is_a_branch", ça me semble plus compréhensible non ? Ensuite le process en fait ce qu'il veut